app/livechat/server/lib/Livechat.js

```diff
@@ -1097,6 +1097,25 @@ export const Livechat = {
 		return true;
 	},
 
+	getRoomMessages({ rid }) {
+		check(rid, String);
+
+		const isLivechat = Promise.await(Rooms.findByTypeInIds('l', [rid])).count();
+
+		if (!isLivechat) {
+			return [];
```

Comment:
I'm wondering if we should throw an error in this case 🤔

Reply:
It does make sense, so I'm throwing the 'invalid-room' error here